pynucastro/networks/rate_collection.py

```diff
@@ -864,7 +895,9 @@ def get_hidden_rates(self):
                 for c in r.get_child_rates():
                     if c.removed:
                         hidden_rates.append(c)
-
+            elif isinstance(r, ModifiedRate):
+                if r.original_rate.removed:
+                    hidden_rates.append(r)
```

Comment:
I think you meant to append `r.original_rate` instead of `r` itself to `hidden_rates`

Reply:
yes, good catch